packages/core/src/lib/banner/__tests__/banner.spec.svelte

```diff
@@ -12,10 +12,9 @@
 
 <script lang="ts">
 import type cx from 'classnames';
-import { Banner } from '$lib';
-import type { BannerVariant } from '$lib/banner.svelte';
+import { Banner, type BannerVariantType } from '$lib';
 
-export let variant: BannerVariant;
+export let variant: BannerVariantType;
```

Comment:
This would be a little breaking change, if we imported this type anywhere. I don't think we do, though